core/src/main/java/nl/nn/adapterframework/http/cxf/SpringSoapBus.java

```diff
@@ -0,0 +1,75 @@
+/*
+   Copyright 2023 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.http.cxf;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.cxf.bus.spring.SpringBus;
+import org.apache.logging.log4j.Logger;
+
+import org.springframework.beans.factory.DisposableBean;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.context.ApplicationContext;
+import org.springframework.web.context.WebApplicationContext;
+
+import nl.nn.adapterframework.util.LogUtil;
+
+class SpringSoapBus extends SpringBus implements InitializingBean, DisposableBean {
+	// This list comes from https://stackoverflow.com/questions/37756464/setting-apache-cxf-bus-properties-for-malicious-xml.
+	// The present code to process them expects them to be integers.
+	private static final List<String> CFX_SECURITY_PROPERTIES = Arrays.asList(
+		"org.apache.cxf.stax.maxAttributeSize",
+		"org.apache.cxf.stax.maxChildElements",
+		"org.apache.cxf.stax.maxElementDepth",
+		"org.apache.cxf.stax.maxAttributeCount",
+		"org.apache.cxf.stax.maxTextLength",
+		"org.apache.cxf.stax.maxElementCount");
+	private static final String SOAP_BUS_PREFIX = "soap.bus.";
+
+	private Logger log = LogUtil.getLogger(this);
+	private WebApplicationContext ctx;
+
+	public SpringSoapBus() {
+		super();
```

Comment:
```suggestion
		super(true);
```
Ik zag dat sinds CXF versie 3.6.0 het handiger is de contructor met true aan te roepen wanneer je de Bus via Spring inlaad.
Zie [deze commit](https://github.com/apache/cxf/commit/2b3ec63eb1c71f9dc58cff5e6244a2dfa356340e) voor meer info.

Reply:
Gedaan.